pkg/cmd/template/cmd.go

```diff
@@ -55,26 +54,20 @@ func NewOptions() *Options {
 	return &Options{}
 }
 
-func NewCmd(o *Options) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:     "template",
-		Aliases: []string{"t", "tpl"},
-		Short:   "Process YAML templates (deprecated; use top-level command -- e.g. `ytt -f-` instead of `ytt template -f-`)",
-		RunE:    func(c *cobra.Command, args []string) error { return o.Run() },
-	}
-	cmd.Flags().BoolVar(&o.IgnoreUnknownComments, "ignore-unknown-comments", false,
+// BindFlags registers top level template flags for template command.
+func (o *Options) BindFlags(cmdFlags CmdFlags) {
```

Comment:
```suggestion
// BindFlags registers template flags for template command.
func (o *Options) BindFlags(cmdFlags CmdFlags) {
```

nit: while this function technically delegates to contained structs for binding their flags, it's whole responsibility is to all flags.